packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -99,14 +99,19 @@ export interface IGCMetadata {
      * - A value greater than 0 means GC is enabled.
      */
     readonly gcFeature?: GCVersion;
-    /** If this is present, the session for this container will expire after this time and the container will close */
-    readonly sessionExpiryTimeoutMs?: number;
     /**
      * Tells whether the GC sweep phase is enabled for this container.
      * - True means sweep phase is enabled.
      * - False means sweep phase is disabled. If GC is disabled as per gcFeature, sweep is also disabled.
      */
-    readonly sweepEnabled?: boolean;
+     readonly sweepEnabled?: boolean;
+
+    /** If this is present, the session for this container will expire after this time and the container will close */
+    readonly sessionExpiryTimeoutMs?: number;
+    /** Maximum duration a snapshot may be cached and then loaded from for this container */
+    readonly maxSnapshotCacheDurationMs?: number | "none";
```

Comment:
Why are there two different types?

Reply:
There are 3 actually:
undefined - old files created when the value was hardcoded to 5 days (or 2 days for even older files)
none - files created at a hypothetical future time where the driver didn't implement the policy so GC shouldn't run
number - files created by this code where the driver does implement the policy and passed in the policy's value